pegasus/routes/v2_user_routes.rb

```diff
@@ -1,45 +0,0 @@
-require 'honeybadger/ruby'
-
-get '/v2/user' do
```

Comment:
I did find one reference to this route: https://github.com/code-dot-org/code-dot-org/blob/1cad5ac667f9ed1a3aa31063e6d06bd21a78e2bd/apps/src/netsim/DashboardUser.js#L75

perhaps possible to preserve `/v2/user` and still delete most everything else in this PR?

Reply:
😱  great catch!